src/Sylius/Bundle/ProductBundle/Resources/config/services/integrations/doctrine/orm.xml

```diff
@@ -14,6 +14,7 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <parameters>
         <parameter key="sylius.repository.product.class">Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductRepository</parameter>
+        <parameter key="sylius.repository.product_association_type.class">Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationTypeRepository</parameter>
```

Comment:
Shouldn't we then remove [this configuration](https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_product.yml#L23)? 🤔

Reply:
You're right 👍

Reply:
@lchrusciel @GSadee Ok with that ?

Reply:
we should :)

Reply:
@lchrusciel @Zales0123 
We forgot something... 
[This line](https://github.com/Sylius/SyliusGridBundle/blob/dfc5f4a0920010550b1af60801179bea7ada5632/src/Bundle/Doctrine/ORM/Driver.php#L48) returns EntityRepository.

However 
```
bin/console debug:container | grep repository.product_association_type
```    
sylius.repository.product_association_type                                                              Sylius\Bundle\ProductBundle\Doctrine\ORM\ProductAssociationTypeRepository                                                      
**sylius.repository.product_association_type_translation**                                                                 Sylius\Bundle\ResourceBundle\Doctrine\ORM\EntityRepository

Reply:
Also, this is the right repository on ResourceController.